### ISSUE TITLE
[BUG] Flutter Texture with HDR

### ISSUE THREAD

**Issue by sudhanshugairola** [jhomlala/betterplayer#1241]
Can you please make this more powerfull with betterplayer because that use flutter_textureview which doesn't provide actual color even colour washout on HDR video.
That will help lots of users.
Developer failed to initialised platform view on Android side to provide a native view.

**Comment by andreasgangso** [comment]
HDR should work fine with https://pub.dev/packages/bccm_player
I just released the package so there may be some setup quirks, but I want the plugin to work for others so just create an issue and I'll help you ASAP.

**Comment by sudhanshugairola** [comment]
Yeah I know HDR work with iOS well but for Android they didn't not platformview with exoplayer library, because exoplayer contains more feature then simple native videoview.

**Comment by liuchuancong** [comment]
@sudhanshugairola  I tried! work bad with Android.

**Comment by jhomlala** [comment]
You're right about the cause here - TextureView doesn't handle HDR color spaces well on Android.

Fixing this means ripping out TextureView and replacing it with PlatformView directly hooked into ExoPlayer. That's a massive architectural rewrite for the entire plugin, and it introduces too many stability risks for standard use cases.

Since we prioritize core stability over new features, we won't be taking this on. For full HDR support on Android, you might want to check out bccm_player (as mentioned in the comments) since it's built around PlatformView.

Closing this out.
